test/datasets/SearchParameters/search.dataset.js

```diff
@@ -355,6 +355,7 @@ function getData() {
     'nbHits': 4,
     'nbPages': 1,
     'page': 0,
+    'params': 'query=&hitsPerPage=20&page=0&facets=%5B%5D&facetFilters=%5B%5B%22city%3AParis%22%2C%22city%3ANew%20York%22%5D%5D',
```

Comment:
Since this PR, `params` is added to SearchResults. Any reason to exclude this? Or let it happen?

Reply:
I don't think it should be removed, there's not really a reason to read it since there's already the SearchParameters class, so we don't need to worry about people abusing this